nucypher/cli/actions.py

```diff
@@ -65,10 +66,17 @@ class UnknownIPAddress(RuntimeError):
     pass
 
 
-def get_password(confirm: bool = False) -> str:
-    keyring_password = os.environ.get("NUCYPHER_KEYRING_PASSWORD", NO_PASSWORD)
+@validate_checksum_address
+def get_client_password(checksum_address) -> str:
+    prompt = f"Enter password to unlock account {checksum_address}"
+    keyring_password = click.prompt(prompt, hide_input=True)
+    return keyring_password
+
+
+def get_nucypher_password(confirm: bool = False, envvar="NUCYPHER_KEYRING_PASSWORD") -> str:
+    keyring_password = os.environ.get(envvar, NO_PASSWORD)
```

Comment:
If this is a boolean var, it might be better to use `cli/config.py:get_env_bool()` here - stricter, but less surprises (the function is perhaps better moved to `utilities` somewhere in that case)

Reply:
You know - I'm thinking of restoring the original state of this code since its not being implemented anywhere.  I had started to do this thinking I was going to share one method for password collection of both client and nucypher keyrings.  Instead, I ended up writing multiple functions.